full-service/src/db/gift_code.rs

```diff
@@ -44,13 +44,7 @@ pub trait GiftCodeModel {
     #[allow(clippy::too_many_arguments)]
     fn create(
         gift_code_b58: &EncodedGiftCode,
-        root_entropy: Option<&RootEntropy>,
-        bip39_entropy: Option<&Vec<u8>>,
-        txo_public_key: &CompressedRistrettoPublic,
         value: i64,
-        memo: &str,
```

Comment:
do we need to update the documentation for this trait now that these args have been removed? looks like the doc still says:
```
    /// Gift code includes:
    /// * entropy
    /// * txo public key
    /// * memo
```

Reply:
nvm. i get that those things are still there, just encoded in the b58